lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -251,6 +253,29 @@ def new_transaction_id
         SecureRandom.uuid
       end
 
+      # Ixopay does not pass any parameters for cardholder/merchant initiated.
+      # Ixopay also doesn't support installment transactions, only recurring
+      # ("RECURRING") and unscheduled ("CARDONFILE").
+      #
+      # Furthermore, Ixopay is slightly unusual in its application of stored
+      # credentials in that the gateway does not return a true
+      # network_transaction_id that can be sent on subsequent transactions.
+
```

Comment:
As always, I'm a fan of these helpful comments! One tiny nitpick is there probably doesn't need to be a blank space on this line, but that may be personal preference.

Reply:
I wasn't sure what the convention was, but happy to get rid of it if that's preferred.

Reply:
It could easily just be my personal preference, and it's honestly not a big deal to me either way :)

So I'm fine if you leave it as-is here, too.

Reply:
It's dead now. 💀